cli/flox/src/commands/containerize.rs

```diff
@@ -8,20 +8,23 @@ use std::str::FromStr;
 use anyhow::{anyhow, Context, Result};
 use bpaf::Bpaf;
 use flox_rust_sdk::flox::Flox;
-use tracing::instrument;
+use tracing::{debug, instrument};
 
 use super::{environment_select, EnvironmentSelect};
 use crate::subcommand_metric;
 use crate::utils::dialog::{Dialog, Spinner};
 use crate::utils::message;
+use crate::utils::openers::first_in_path;
 
 // Containerize an environment
 #[derive(Bpaf, Clone, Debug)]
 pub struct Containerize {
     #[bpaf(external(environment_select), fallback(Default::default()))]
     environment: EnvironmentSelect,
 
-    /// Output to write the container to, defaults to '{environment}-container.tar'
+    /// Output to write the container to.
+    /// Defaults to loading the container into a supported container runtime if available
```

Comment:
nit: a container runtime is something like containerd, CRI-O, or Docker engine (https://kubernetes.io/docs/setup/production-environment/container-runtimes/). I think it would be more correct to describe this as
```
Defaults to loading into container storage if docker or podman is present or otherwise writes to a file '{name}-container.tar'
```

Same comment wherever we use the word `runtime`

Reply:
When runtime is the thing below docker and podman what category are docker and podman then?

Reply:
A container management tool? Idk. I'm not sure if they have a well understood name, so I think it's clearer just to mention the tools we support